.jenkins/Jenkinsfile

```diff
@@ -203,6 +203,17 @@ pipeline {
             }
           }
         }
+        stage('Windows Debug') {
+           agent {
+             node {
+               label 'SGXFLC-Windows'
+             }
+
+           }
+           steps {
+             bat '''mkdir build && cd build && cmake -G "Visual Studio 15 2017 Win64" .. && pushd . && "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\Common7\\Tools\\LaunchDevCmd.bat" && popd && cmake --build . --config Debug && ctest -C Debug'''
```

Comment:
Nonblocking: consider making this a separate script file like `test-build-config` and simplifying where appropriate (e.g. why not just do everything in the dev shell instead of `pushd` & `popd` across the shell transition?)